ansible/roles/rust/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+- name: install rustup, build-essential and musl
+  ansible.builtin.shell: |
+    #!/bin/bash
+    apt update -y
+    DEBIAN_FRONTEND=noninteractive apt install build-essential musl musl-tools -y
+    curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh  -s -- --default-toolchain stable -y
+    . ${HOME}/.cargo/env
```

Comment:
Currently getting errors without sourcing the cargo env variable and exporting PATH environment variable.

`TASK [rust : install rustup] ******************************************************************************************************************************************************************************
fatal: [angus-wll3-build]: FAILED! => {"changed": true, "cmd": "#!/bin/bash\ncurl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh  -s -- --default-toolchain stable -y\nrustup target add x86_64-unknown-linux-musl", "delta": "0:00:20.321900", "end": "2023-07-12 20:49:34.148253", "msg": "non-zero return code", "rc": 127, "start": "2023-07-12 20:49:13.826353", "stderr": "info: downloading installer\ninfo: profile set to 'default'\ninfo: default host triple is x86_64-unknown-linux-gnu\ninfo: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'\ninfo: latest update on 2023-06-01, rust version 1.70.0 (90c541806 2023-05-31)\ninfo: downloading component 'cargo'\ninfo: downloading component 'clippy'\ninfo: downloading component 'rust-docs'\ninfo: downloading component 'rust-std'\ninfo: downloading component 'rustc'\ninfo: downloading component 'rustfmt'\ninfo: installing component 'cargo'\ninfo: installing component 'clippy'\ninfo: installing component 'rust-docs'\ninfo: installing component 'rust-std'\ninfo: installing component 'rustc'\ninfo: installing component 'rustfmt'\ninfo: default toolchain set to 'stable-x86_64-unknown-linux-gnu'\n/bin/sh: 3: rustup: not found", "stderr_lines": ["info: downloading installer", "info: profile set to 'default'", "info: default host triple is x86_64-unknown-linux-gnu", "info: syncing channel updates for 'stable-x86_64-unknown-linux-gnu'", "info: latest update on 2023-06-01, rust version 1.70.0 (90c541806 2023-05-31)", "info: downloading component 'cargo'", "info: downloading component 'clippy'", "info: downloading component 'rust-docs'", "info: downloading component 'rust-std'", "info: downloading component 'rustc'", "info: downloading component 'rustfmt'", "info: installing component 'cargo'", "info: installing component 'clippy'", "info: installing component 'rust-docs'", "info: installing component 'rust-std'", "info: installing component 'rustc'", "info: installing component 'rustfmt'", "info: default toolchain set to 'stable-x86_64-unknown-linux-gnu'", "/bin/sh: 3: rustup: not found"], "stdout": "\n  stable-x86_64-unknown-linux-gnu installed - rustc 1.70.0 (90c541806 2023-05-31)\n\n\nRust is installed now. Great!\n\nTo get started you may need to restart your current shell.\nThis would reload your PATH environment variable to include\nCargo's bin directory ($HOME/.cargo/bin).\n\nTo configure your current shell, run:\nsource \"$HOME/.cargo/env\"", "stdout_lines": ["", "  stable-x86_64-unknown-linux-gnu installed - rustc 1.70.0 (90c541806 2023-05-31)", "", "", "Rust is installed now. Great!", "", "To get started you may need to restart your current shell.", "This would reload your PATH environment variable to include", "Cargo's bin directory ($HOME/.cargo/bin).", "", "To configure your current shell, run:", "source \"$HOME/.cargo/env\""]}`